cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -470,12 +470,22 @@ def test_datastore_relationships(setup_provider, testing_vm):
     view.flash.assert_no_error()
 
 
+@pytest.fixture(scope="function")
+def cluster(provider):
+    collection = provider.appliance.collections.clusters
+    cluster_name = provider.data["cap_and_util"]["cluster"]
```

Comment:
I would check exception as data cumming from data yaml. may be tag not available in this condition we need to skip test.